### PR TITLE
test: improve video label snapshots

### PR DESCRIPTION
**Pull request by colincclark**

### CODE REVIEW

File: packages/video-label/__tests__/video-label-style.native.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import VideoLabel from "../src/video-label";
+
+export default () => {
+  addSerializers(
+    expect,
+    enzymeTreeSerializer(),
+    compose(print, minimalNativeTransform, flattenStyleTransform)
+  );
+
+  it("1. video label with a title", () => {
```
Comment (craigbilner) [inline packages/video-label/__tests__/video-label-style.native.js]: iterator iterator iterator

File: packages/video-label/__tests__/web/__snapshots__/video-label-style.web.test.js.snap [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+        <div
+          className="S1"
+        >
+          <IconVideo
+            fillColour="black"
+            height={8}
+          />
+        </div>
+      </View>
+      <Text
```
Comment (craigbilner) [inline packages/video-label/__tests__/web/__snapshots__/video-label-style.web.test.js.snap]: remove native

Comment (colincclark) [reply packages/video-label/__tests__/web/__snapshots__/video-label-style.web.test.js.snap]: done

File: packages/video-label/__tests__/video-label-style.native.js (hunk 1 above)
Comment (colincclark) [reply packages/video-label/__tests__/video-label-style.native.js]: done

File: packages/video-label/__tests__/web/__snapshots__/video-label.web.test.js.snap [hunk 3]
```diff
@@ -1,75 +1,43 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`Video Label test on web renders VideoLabel with an explicit title 1`] = `
-<div>
-  <div>
-    <svg
+exports[`1. video label with a title 1`] = `
+<View>
```
Comment (craigbilner) [inline packages/video-label/__tests__/web/__snapshots__/video-label.web.test.js.snap]: shouldn't have the native wrappers here, can either "melt" them or use the enzyme renderered serializer instead

File: packages/video-label/__tests__/web/__snapshots__/video-label-with-style.web.test.js.snap [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+}
+</style>
+
+<div
+  className="S3"
+>
+  <div
+    className="S1"
+  >
+    <svg
```
Comment (craigbilner) [inline packages/video-label/__tests__/web/__snapshots__/video-label-with-style.web.test.js.snap]: ideally this should also be `IconVideo`?

Comment (colincclark) [reply packages/video-label/__tests__/web/__snapshots__/video-label-with-style.web.test.js.snap]: done

File: packages/video-label/__tests__/web/__snapshots__/video-label.web.test.js.snap (hunk 3 above)
Comment (colincclark) [reply packages/video-label/__tests__/web/__snapshots__/video-label.web.test.js.snap]: done

**Pull request merged**